index.html

```diff
@@ -147,9 +98,58 @@ <h1>Hello<span class="wave">👋🏾</span>
 				</div>
 			</div>
 		</div>
+		<!-- Added Style Tag to center "Let's Get Started" button in the Modal here: -->
```

Comment:
I see here where you moved all of the animated Modal functionalities within the same space in the HTML doc. However, I'm not seeing the div class for the modal activation or the style tags for the modal content. Was that portion omitted from the changes you made which is why it isn't showing up here for this PR? Just wanting to clarify.

Reply:
Good question! When going through the HTML, I believe that's the code written on line 64 - is that what you're referring to? I didn't move any of those lines, but I see why it would look alarming if it wasn't where you were expecting to see it.